common/src/main/scala/hmda/model/filing/lar/enums/AutomatedUnderwritingResultEnum.scala

```diff
@@ -124,6 +131,42 @@ case object AutomatedUnderwritingResultNotApplicable
   override val description: String = "Not applicable"
 }
 
+case object AcceptEligible extends AutomatedUnderwritingResultEnum {
```

Comment:
Looks like we both needed to update the ENUMs for AUR. @kgudel  Once this gets merged in I'll update my PR to reflect what you have here.

Reply:
Yeah, we ended up both needing that piece :)